File: back/students/src/main/java/doczilla/testing/students/model/Student.java
package doczilla.testing.students.model;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDate;

@Getter
@Builder
public class Student {
    private int id;
    private String name;
    private String surname;
    private String patronymic;
    private LocalDate birthday;
    private String groupName;
}

File: front/script/model/Student.js
export class Student {
    constructor(id, name, surname, patronymic, birthday, groupName) {
        this.id = id;
        this.name = name;
        this.surname = surname;
        this.patronymic = patronymic;
        this.birthday = birthday;
        this.groupName = groupName;
    }

    getJson() {
        return {
            name: this.name,
            surname: this.surname,
            patronymic: this.patronymic,
            birthday: this.birthday,
            groupName: this.groupName
        };
    }
}